src/fidesctl/cli/commands/export.py

```diff
@@ -1,7 +1,11 @@
 """Contains the export group of CLI commands for Fidesctl."""
 import click
 
-from fidesctl.cli.options import dry_flag, manifests_dir_argument
+from fidesctl.cli.options import (  # organization_fides_key_argument,
```

Comment:
I think this comment was supposed to get removed, but I could be missing something else

Reply:
ya should be removed, thank you 👍🏽 
I spoke with @earmenda briefly about this too during a 1:1 and am thinking about replacing the other positional argument with an option flag as well (output directory) to help clean up how the command feels